docs/rst/installation/docker.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: ../exports/alias.include
+
+.. _docker:
+
+############
+Docker image
+############
+
+A DDS Router Docker image is distributed with Ubuntu 20.04 base image.
+This image launches an instance of DDS Router that is configured using a *YAML* configuration file provided by the user
+and shared with the Docker container.
+The steps to run DDS Router in a Docker container are explained below.
+
+#.  Download the compressed Docker image in ``.tar`` format from the
```

Comment:
Is there any way to add a line break between lines in this list? The page seems cramped.